Dockerfile

```diff
@@ -13,18 +10,16 @@ ENV ENV_API_KEY=$API_KEY
 ARG CONTROLLER_URL
 ENV ENV_CONTROLLER_URL=$CONTROLLER_URL
 
+# e.g True or False
+ARG STORE_UUID=False
+ENV ENV_STORE_UUID=$STORE_UUID
 
-
-# Download certificate and key from the customer portal (https://cs.nginx.com)
+# Download certificate (nginx-repo.crt) and key (nginx-repo.key) from the customer portal (https://cs.nginx.com)
 # and copy to the build context
-RUN mkdir -p /etc/nginx/sites-enabled/ 
-COPY $cert /etc/ssl/nginx/
-COPY $key /etc/ssl/nginx/
-COPY stub_status.conf /etc/nginx/sites-enabled/ 
+COPY nginx-repo.* /etc/ssl/nginx/
```

Comment:
It would be great if there was at least a comment here referencing that we need both the repo certificate and key.
I think that the COPY of nginx-repo.* kind of hides or masks this detail for folks that don't already know that.
I don't deny that a single step is more efficient.

Reply:
Look at the Dockerfile itself.  I did add a comment about what files to grab.

Reply:
`# Download certificate (nginx-repo.crt) and key (nginx-repo.key) from the customer portal (https://cs.nginx.com)
# and copy to the build context`

Reply:
Thanks.  Skipped right over that.